pkg/render/typha.go

```diff
@@ -451,7 +451,7 @@ func (c *typhaComponent) typhaEnvVars() []v1.EnvVar {
 	}
 	if c.cr.Spec.Variant == operator.TigeraSecureEnterprise {
 		extraTyphaEnv := []v1.EnvVar{
-			// When we add AWS integration then we need Security group stuff here
+		// When we add AWS integration then we need Security group stuff here
```

Comment:
whooops someone finally touched these lines.

We should just delete this section - what's the point of having an empty `extraTyphaEnv` that is a placeholder for future work?